### PR TITLE
LibGfx: Let Painter care about TinyVG transforms

### PR DESCRIPTION
**Pull request by shlyakpavel**
This helps us with non-uniform scales, and makes things simple

### CODE REVIEW

File: Libraries/LibGfx/ImageFormats/TinyVGLoader.cpp [hunk 1]
```diff
@@ -473,27 +473,23 @@ ErrorOr<NonnullRefPtr<TinyVGDecodedImageData>> TinyVGDecodedImageData::decode(St
 
 void TinyVGDecodedImageData::draw_transformed(Painter& painter, AffineTransform transform) const
 {
-    // FIXME: Correctly handle non-uniform scales.
-    auto scale = max(transform.x_scale(), transform.y_scale());
+    painter.set_transform(transform);
```
[the remark this answers is not in the document]
Comment (gmta) [reply Libraries/LibGfx/ImageFormats/TinyVGLoader.cpp:476]: I.e. maybe this should go into `VectorGraphic` instead, and rework the `draw_transformed` methods?

[the remark this answers is not in the document]
Comment (shlyakpavel) [reply Libraries/LibGfx/ImageFormats/TinyVGLoader.cpp:476]: Within this codebase and this PR, Painter has just been created for a bitmap, so it cannot hold another transformation

[the remark this answers is not in the document]
Comment (shlyakpavel) [reply Libraries/LibGfx/ImageFormats/TinyVGLoader.cpp:476]: Agreed

[the remark this answers is not in the document]
Comment (shlyakpavel) [reply Libraries/LibGfx/ImageFormats/TinyVGLoader.cpp]: Done

**Review by albuera1987 — APPROVED** [review]

**Comment by shlyakpavel** [comment]
Rebased on master just in case

File: Libraries/LibGfx/ImageFormats/TinyVGLoader.cpp [hunk 2]
```diff
@@ -471,29 +471,24 @@ ErrorOr<NonnullRefPtr<TinyVGDecodedImageData>> TinyVGDecodedImageData::decode(St
...
-    // FIXME: Correctly handle non-uniform scales.
-    auto scale = max(transform.x_scale(), transform.y_scale());
     for (auto const& command : draw_commands()) {
-        auto draw_path = command.path.copy_transformed(transform);
+        auto draw_path = command.path;
         if (command.fill.has_value()) {
-            auto fill_path = draw_path;
-            fill_path.close_all_subpaths();
+            draw_path.close_all_subpaths();
             command.fill->visit(
```
Comment (AtkinsSJ) [inline Libraries/LibGfx/ImageFormats/TinyVGLoader.cpp:480]: This is subtly different behaviour than before: Previously, the closed path would be used for the fill, but the potentially-unclosed path for the stroke. Now you use the closed path for both.

Comment (shlyakpavel) [reply Libraries/LibGfx/ImageFormats/TinyVGLoader.cpp:480]: You're right let me fix that

Comment (shlyakpavel) [reply Libraries/LibGfx/ImageFormats/TinyVGLoader.cpp:481]: Done

**Pull request merged**